packages/lambda-tiler/src/routes/health.ts

```diff
@@ -40,6 +84,77 @@ export async function updateExpectedTile(test: TestTile, newTileData: Buffer, di
   await fs.promises.writeFile(`${expectedFileName}.diff.png`, imgPng);
 }
 
+/**
+ * Compare and validate the raster test tile from server with pixel match
+ */
+async function validateRasterTile(
+  tileSet: ConfigTileSetRaster,
+  test: TestTile,
+  req: LambdaHttpRequest,
+): Promise<LambdaHttpResponse | undefined> {
+  // Get the parse response tile to raw buffer
+  const response = await TileXyzRaster.tile(req, tileSet, test);
+  if (response.status !== 200) return new LambdaHttpResponse(500, response.statusDescription);
+  if (!Buffer.isBuffer(response._body)) throw new LambdaHttpResponse(500, 'Not a Buffer response content.');
+  const resImgBuffer = await Sharp(response._body).raw().toBuffer();
+
+  // Get test tile to compare
+  const testBuffer = await getTestBuffer(test);
+  test.buf = testBuffer;
+  const testImgBuffer = await Sharp(testBuffer).raw().toBuffer();
+
+  const outputBuffer = Buffer.alloc(testImgBuffer.length);
+  const missMatchedPixels = PixelMatch(testImgBuffer, resImgBuffer, outputBuffer, TileSize, TileSize);
+  if (missMatchedPixels) {
+    /** Uncomment this to overwite the expected files */
+    // await updateExpectedTile(test, response._body as Buffer, outputBuffer);
+    req.log.error({ missMatchedPixels, projection: test.tileMatrix.identifier, xyz: test.tile }, 'Health:MissMatch');
+    return new LambdaHttpResponse(500, 'TileSet does not match.');
+  }
+  return;
+}
+
+function assertLayer(tile: VectorTile, testFeature: TestFeature): boolean {
+  const layer = tile.layers[testFeature.layer];
+  for (let i = 0; i < layer.length; i++) {
+    const feature = layer.feature(i);
+    if (feature.properties[testFeature.property] === testFeature.value) return true;
+  }
+  return false;
+}
+
+/**
+ * Check the existence of a feature property in side the vector tile
+ */
+function featureCheck(tile: VectorTile, testTile: TestTile): LambdaHttpResponse | undefined {
```

Comment:
this never returns anything?

```suggestion
function featureCheck(tile: VectorTile, testTile: TestTile): undefined {
```